plugin/trino-phoenix/src/main/java/io/trino/plugin/phoenix/MetadataUtil.java

```diff
@@ -24,18 +24,18 @@
 {
     private MetadataUtil() {}
 
-    public static String getEscapedTableName(Optional<String> schema, String table)
+    public static @Nullable String getEscapedTableName(@Nullable String schema, String table)
     {
-        return SchemaUtil.getEscapedTableName(toPhoenixSchemaName(schema).orElse(null), table);
+        return SchemaUtil.getEscapedTableName(toPhoenixSchemaName(schema), table);
     }
 
-    public static Optional<String> toPhoenixSchemaName(Optional<String> trinoSchemaName)
+    public static @Nullable String toPhoenixSchemaName(@Nullable String trinoSchemaName)
     {
-        return trinoSchemaName.map(schemaName -> DEFAULT_SCHEMA.equalsIgnoreCase(schemaName) ? NULL_SCHEMA_NAME : schemaName);
+        return DEFAULT_SCHEMA.equalsIgnoreCase(trinoSchemaName) ? NULL_SCHEMA_NAME : trinoSchemaName;
```

Comment:
Does it make sense to return `null` from here? If `trinoSchemaName` should we return `null` or `NULL_SCHEMA_NAME`?

Reply:
My guess is that the names can't really be `null` and these nullity annotations are overshooting a bit here.

Reply:
> Does it make sense to return null from here?  

If it was a `null` then it make sense. API suggests it could be null in general, but in Phoenix I don't think it is a real case. It is a overshooting, but I wasn't 100% sure.